dart/lib/src/http_client/failed_request_client.dart

```diff
@@ -183,6 +186,15 @@ class FailedRequestClient extends BaseClient {
       throwable: throwableMechanism,
       request: sentryRequest,
     );
+
+    if (response != null) {
```

Comment:
Not available in the response object. BTW cookies are not really a standalone part of an http response - they're sent as "set-cookie" headers

Reply:
You can read from the headers, https://github.com/getsentry/sentry-dart/blob/d7758e8c1a1884d46734ccb197dbcde683bc4189/dart/lib/src/http_client/failed_request_client.dart#L167
IF the response one is called `Set-Cookie`, just use this one for the response.

Reply:
Yep, that's what I meant in the BTW part of my comment. 

Wouldnt it be better to do this on the server instead of each client lib?

Reply:
The server already does that actually, but it's a back compatibility issue with older versions of the server.

Reply:
I've checked the http response and it's providing headers as a map in which case it gives all the cookies as a single string, something like: `AWSALB=AWSALB_TEST; Expires=Tue, 26 Apr 2022 00:26:55 GMT; Path=/,AWSALBCORS=AWSALBCORS_TEST; Expires=Tue, 26 Apr 2022 00:26:55 GMT; Path=/; SameSite=None; Secure,jwt_token=JWT_TEST; Domain=.test.com; Max-Age=31536000; Path=/; expires=Wed, 19-Apr-2023 00:26:55 GMT; SameSite=lax; Secure,csrf_token=CSRF_TOKEN_TEST_1; Domain=.test.com; Max-Age=31536000; Path=/; expires=Wed, 19-Apr-2023 00:26:55 GMT,csrf_token=CSRF_TOKEN_TEST_2; Domain=.test.com; Max-Age=31536000; Path=/; expires=Wed, 19-Apr-2023 00:26:55 GMT,wuuid=WUUID_TEST`

Parsing that correctly is non-trivial and I'd move it out to a separate issue, prioritized according to the actual user needs (might be pretty low since you're saying it's just for backward compatibility).

Also see https://github.com/dart-lang/http/issues/24 - though any change in that area is unlikely to happen any time soon. I'm just flabbergasted how anyone could have implemented headers as a `map<string,string>` and called it a day, resulting in a package that can't move away from it due to compatibility and being one of the core ones basically.

Reply:
We don't need to parse it, `cookies` is a String.
The server actually accepts multiple types, See https://develop.sentry.dev/sdk/event-payloads/types/#request for `cookies`, same for the response object.
For strongly typed languages, we have to choose one and it's being String for some reason, so I'd not care about the format, the server will accept and display it nicely anyway.
`response.headers['Set-Cookie']` will do.

Reply:
added in https://github.com/getsentry/sentry-dart/pull/1095/commits/fa59ae2357be26055cf25f8601111c57e8c85c2f